SwitchSpeak-iOS/SwitchSpeak-iOS/ViewController.swift

```diff
@@ -10,27 +10,62 @@ import UIKit
 class ViewController: UIViewController {
 	@IBOutlet weak var TapButton: UIButton!
 	var breadcrumbs = CrumbStack()
-    var touchGrid:TouchGrid?
+    var touchSelection:TouchSelection?
 	
 	override func viewDidLoad() {
 		super.viewDidLoad()
-        
+		//	dummy array of phrases for the first grid selection
+		let dummyPhrases: [String] = ["1", "2", "3", "4", "5", "6", "7", "8","9","10","11"]
```

Comment:
there are a couple of such dummy arrays....we can update these once the database is set up.